resources/migrations/001_update_migrations.yaml

```diff
@@ -9270,6 +9270,25 @@ databaseChangeLog:
                   type: ${text.type}
                   remarks: running, failed, or completed
 
+  - changeSet:
+      id: v51.2024-09-03T16:54:18
+      author: adam-james
+      comment: Add pivot_results to pulse_card
+      preConditions:
+        - not:
+            - columnExists:
+                tableName: pulse_card
+                columnName: pivot_results
+      changes:
+        - addColumn:
+            tableName: pulse_card
+            columns:
+              - column:
+                  name: pivot_results
+                  type: ${boolean.type}
+                  defaultValue: false
+                  remarks: Whether or not to apply pivot processing to the rows of the export
```

Comment:
Do we need to add this column? This info is already stored in viz settings but honestly it feels like it could be part of the query itself

Reply:
Why do we only need to add it to `pulse_card` and not to `report_card`?